recipes/otsvm/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DBUILD_DOC=OFF \
+  ..
+
+make install -j${CPU_COUNT}
+DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib ctest -R pyinstallcheck -E saveload --output-on-failure -j${CPU_COUNT}
```

Comment:
@jschueller is there a test we can run? Like `make check` or `ctest`? If not this is good to go.